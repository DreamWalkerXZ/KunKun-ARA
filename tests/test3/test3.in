10 10 15
0 0 1 1 0 0 1 1 0 0 
1 0 1 1 1 1 0 0 0 1 
0 0 0 0 1 0 0 1 0 0 
0 1 1 0 1 0 0 1 0 0 
0 1 0 0 1 0 0 1 0 0 
0 0 1 0 0 1 0 1 0 0 
1 0 0 1 1 1 0 1 0 0 
0 0 0 0 0 1 1 0 0 1 
1 1 1 1 0 0 0 1 1 1 
0 0 0 0 0 1 0 0 0 0 
3
13 0 5 
11 3 8 
10 5 6 
10
12
11
9
8
7
5
4
3
2
1
