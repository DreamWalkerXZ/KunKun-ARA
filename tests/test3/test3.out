11
3 3 4 3 5 4 4 5 5 6 6 6 7 7 8 6 9 7 9 8 9 9 
10
4 3 5 4 4 5 5 6 6 6 7 7 8 6 9 7 9 8 9 9 
9
5 3 6 2 7 3 8 4 8 5 9 6 9 7 9 8 9 9 
8
6 2 7 3 8 4 8 5 9 6 9 7 9 8 9 9 
7
7 3 8 4 8 5 9 6 9 7 9 8 9 9 
5
8 5 9 6 9 7 9 8 9 9 
4
9 6 9 7 9 8 9 9 
3
9 7 9 8 9 9 
2
9 8 9 9 
15
0 0 1 1 2 2 3 3 4 3 5 4 5 3 6 2 7 3 8 4 8 5 9 6 9 7 9 8 9 9 
